### PR TITLE
ENH: add minimum_bounding_radius as GeoSeries method

### PR DESCRIPTION
**Pull request by anastassiavybornova**

### CODE REVIEW

File: geopandas/tests/test_geom_methods.py [hunk 1]
```diff
@@ -1466,3 +1466,36 @@ def test_get_coordinates_not(self):
             NotImplementedError, match="shapely >= 2.0 or PyGEOS are required"
         ):
             self.g11.get_coordinates()
+
+    def test_minimum_bounding_radius(self):
```
Comment (martinfleis) [inline geopandas/tests/test_geom_methods.py:1470]:
```suggestion
    @pytest.mark.skipif(
        not (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
        reason="minimum_bounding_radius not implemented for shapely<2",
    )
    def test_minimum_bounding_radius(self):
```

You need to skip this test for CI environments that still use shapely 1.8 (for backwards compatibility). And as the `test_get_coordinates_not`, you should write a tiny another one flipping the condition and checking the error.

File: geopandas/tests/test_geom_methods.py [hunk 2]
```diff
@@ -1466,3 +1466,36 @@ def test_get_coordinates_not(self):
...
+# def test_minimum_bounding_circle(self):
+#     mbc = self.g1.minimum_bounding_circle()
+#     centers = GeoSeries([Point(0.5, 0.5)] * 2)
+#     assert np.all(mbc.centroid.geom_equals_exact(centers, 0.001))
+#     assert_series_equal(
+#         mbc.area,
+#         Series([1.560723, 1.560723]),
+#     )
+#     assert isinstance(mbc, GeoSeries)
+#     assert self.g1.crs == mbc.crs
```
Comment (martinfleis) [inline geopandas/tests/test_geom_methods.py:1501]:
```suggestion
```

**Review by martinfleis — COMMENTED** [review]
Looking good!

File: geopandas/tests/test_geom_methods.py (hunk 1 above)
Comment (anastassiavybornova) [reply geopandas/tests/test_geom_methods.py:1474]: added!

File: geopandas/tests/test_geom_methods.py [hunk 3]
```diff
@@ -1466,3 +1466,30 @@ def test_get_coordinates_not(self):
...
+            mbr_lines,
+            Series([0.707106, 0.707106]),
+        )
+
+    @pytest.mark.skipif(
+        (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
+        reason="minimum_bounding_radius not implemented for shapely<2",
+    )
+    def test_minimium_bounding_radius_not(self):
+        with pytest.raises(NotImplementedError, match="shapely >= 2.0 is required"):
```
Comment (martinfleis) [inline geopandas/tests/test_geom_methods.py:1494]:
```suggestion
        with pytest.raises(NotImplementedError, match="shapely >= 2.0 or PyGEOS is required"):
```

This is the correct match.

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,7 +12,8 @@ New features and improvements:
 - Support for sorting geometries (for example, using ``sort_values()``) based on
   the distance along the Hilbert curve (#2070).
 - Added ``minimum_bounding_circle()`` method from shapely to GeoSeries/GeoDataframe (#2621).
-- Support specifying ``min_zoom`` and ``max_zoom`` inside the ``map_kwds`` argument for ``.explore()`` (#2599).
+- Support specifying ``min_zoom`` and ``max_zoom`` inside the ``map_kwds`` argument for ``.explore()`` (#2599)
```
Comment (martinfleis) [inline CHANGELOG.md:15]:
```suggestion
- Support specifying ``min_zoom`` and ``max_zoom`` inside the ``map_kwds`` argument for ``.explore()`` (#2599).
```
Adding the period back :).

**Review by martinfleis — APPROVED** [review]
Thanks! I committed two minor details. Looks good to me!

File: geopandas/tests/test_geom_methods.py [hunk 5]
```diff
@@ -1466,3 +1466,30 @@ def test_get_coordinates_not(self):
...
+            mbr_lines,
+            Series([0.707106, 0.707106]),
+        )
+
+    @pytest.mark.skipif(
+        (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
+        reason="minimum_bounding_radius not implemented for shapely<2",
+    )
+    def test_minimium_bounding_radius_not(self):
+        with pytest.raises(NotImplementedError, match="shapely >= 2.0 or PyGEOS is required"):
```
Comment (martinfleis) [inline geopandas/tests/test_geom_methods.py:1494]:
```suggestion
        with pytest.raises(
            NotImplementedError, match="shapely >= 2.0 or PyGEOS is required"
        ):
```

**Review by m-richards — APPROVED** [review]

**Comment by m-richards** [comment]
Thanks!

**Pull request merged**